statefun-kafka-io/src/main/java/org/apache/flink/statefun/sdk/kafka/KafkaIngressBuilder.java

```diff
@@ -39,8 +39,7 @@
   private Class<? extends KafkaIngressDeserializer<T>> deserializerClass;
   private String kafkaAddress;
   private KafkaIngressAutoResetPosition autoResetPosition = KafkaIngressAutoResetPosition.LATEST;
-  private KafkaIngressStartupPosition startupPosition =
-      KafkaIngressStartupPosition.fromGroupOffsets();
+  private KafkaIngressStartupPosition startupPosition = KafkaIngressStartupPosition.fromLatest();
```

Comment:
Why latest? I think earliest is more intuitive (and also the default on the connector)

Reply:
The `FlinkKafkaConsumer` actually uses committed group offsets as the default:
https://github.com/apache/flink/blob/master/flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumerBase.java#L159

The reason we chose to change this to latest, is because group offsets startup require that the user must set the consumer group id. We wanted to go with a default that requires the minimal setup to get started with using the ingress.

Reply:
As for latest v.s. earliest, for a typical Kafka consumer (non-Flink/StateFun), when consuming from a topic that has no previous record of committed offsets, that's the default behaviour AFAIK.

See the section about `auto.offset.reset` in the Kafka docs here:
https://docs.confluent.io/current/clients/consumer.html#id1

Reply:
Sounds good to me, I don't have a strong opinion on this one